### ISSUE TITLE
Document for new API

### ISSUE THREAD

**Issue by volethanh** [houseofcat/RabbitMQ.Core#1]
Hi,

I've used CookedRabbit and now I saw you moved to RabbitMQ.Core as a .net core project. But the APIs changed completely, so you use document for all new APIs. 

The project Benchmark not include Consumer benchmark

Thanks,
Thanh

**Comment by houseofcat** [comment]
Yep I will be adding new documentation. It's even easier but it is a work in progress. Its been slow going because of holiday season but it will pick back up.

**Comment by aygkhn** [comment]
we are waiting new documentation.

**Comment by houseofcat** [comment]
This repo is no longer worked on. New work moved to GitHub.com/houseofcat/Tesseract.

**Comment by aygkhn** [comment]
> This repo is no longer worked on. New work moved to GitHub.com/houseofcat/Tesseract.

thanks.
